unfoldable.cabal

```diff
@@ -42,7 +42,7 @@ library
       base         >= 4   && < 5
     , containers   >= 0.5 && < 0.7
     , transformers >= 0.4 && < 0.6
-    , random       >= 1.0 && < 1.2
+    , random       >= 1.2.0 && < 1.3
```

Comment:
Thanks for taking the time to do this! One question: did you mean to change the lower bound here?

Reply:
This was my mistake, sorry - I thought [this breaking change in the changelog](https://hackage.haskell.org/package/random-1.2.1.1/changelog#120) was relevant, but I don't think it actually is! I'll update this MR.